src/test/java/io/thinkit/edc/client/connector/ApplicationObservabilityTest.java

```diff
@@ -38,139 +38,65 @@ void should_check_health() {
         });
     }
 
-    @Test
-    void should_check_health_async() {
-        try {
-            var result = applicationObservability.checkHealthAsync().get();
-            assertThat(result.isSucceeded()).isTrue();
-            assertThat(result.getContent().isSystemHealthy()).isTrue();
-            assertThat(result.getContent().componentResults())
-                    .isNotNull()
-                    .first()
-                    .satisfies(componentResult -> {
-                        assertThat(componentResult.component()).isNotNull().isEqualTo("string");
-                        assertThat(componentResult.isHealthy()).isTrue();
-                        assertThat(componentResult.failure()).isNotNull().satisfies(failure -> {
-                            assertThat(failure.failureDetail()).isNotNull().isEqualTo("string");
-                            assertThat(failure.messages().size()).isGreaterThan(0);
-                        });
-                    });
-        } catch (InterruptedException | ExecutionException e) {
-            throw new RuntimeException(e);
+    @Nested
+    class Sync {
+        @Test
+        void should_check_health() {
+            var result = applicationObservability.checkHealth();
+            health_status_response(result);
```

Comment:
is not clear that this is an assertion, please use the assertj way to represent it:
```assertThat(result).satisfies(ApplicationObservabilityTest.this::health_status_response)```
same for all the other "sync" tests